boa3_test/tests/examples_tests/test_nep11_non_divisible.py

```diff
@@ -25,500 +32,520 @@ class TestNEP11Template(BoaTest):
+        result, _ = await self.call('balanceOf', [owner_account], return_type=int)
+        self.assertEqual(expected, result)
+
+        bad_account = bytes(10)
+        with self.assertRaises(boatestcase.AssertException) as context:
+            await self.call("balanceOf", [bad_account], return_type=int)
+        self.assertEqual(str(context.exception), 'Not a valid address')
+
+        bad_account = bytes(30)
+        with self.assertRaises(boatestcase.AssertException) as context:
+            await self.call("balanceOf", [bad_account], return_type=int)
+        self.assertEqual(str(context.exception), 'Not a valid address')
+
+    async def test_tokens_of(self):
+        no_balance_account = types.UInt160.zero()
+        # TODO: #86drqwhx0 neo-go in the current version of boa-test-constructor is not configured to return Iterators
+        with self.assertRaises(ValueError) as context:
+            result, _ = await self.call('tokensOf', [no_balance_account], return_type=list)
+            self.assertEqual([], result)
+
+        self.assertRegex(str(context.exception), 'Interop stack item only supports iterators')
+
+        tokens_of_storage = await self.get_storage(
+            self.ACCOUNT_PREFIX + no_balance_account.to_array(),
+            remove_prefix=True
+        )
+        self.assertEqual(0, len(tokens_of_storage))
+
+        # TODO: #86drqwhx0 neo-go in the current version of boa-test-constructor is not configured to return Iterators
+        with self.assertRaises(ValueError) as context:
+            result, _ = await self.call('tokensOf', [self.owner.script_hash], return_type=list)
+            self.assertEqual(self.OWNER_BALANCE, len(result))
+
+        self.assertRegex(
+            str(context.exception),
+            fr"item is not of type 'StackItemType.\w+' but of type '{StackItemType.INTEROP_INTERFACE}'"
+        )
+
+        tokens_of_storage = await self.get_storage(
+            self.ACCOUNT_PREFIX + self.owner.script_hash.to_array(),
+            remove_prefix=True
+        )
+        self.assertEqual(self.OWNER_BALANCE, len(tokens_of_storage))
+
+    async def test_transfer_success(self):
+        token = self.TOKEN_ID_TRANSFER_TEST
+        from_account = self.account1.script_hash
+        to_account = self.account2.script_hash
+
+        total_supply, _ = await self.call('totalSupply', [], return_type=int)
 
         # check owner before
-        invokes.append(runner.call_contract(path, 'ownerOf', token))
-        expected_results.append(other_account_1_script_hash)
+        result, _ = await self.call('ownerOf', [token], return_type=types.UInt160)
+        self.assertEqual(from_account, result)
 
         # transfer
-        invokes.append(runner.call_contract(path, 'transfer',
-                                            other_account_2_script_hash, token, None,
-                                            expected_result_type=bool))
-        expected_results.append(True)
+        result, _ = await self.call(
```

Comment:
I think it would be nice to also check the Nep11TransferEvent notification here. You already check it later when burning and minting, but there isn't one test from one account to another.